src/ProductConstructionService/ProductConstructionService.Cli/Operations/DeploymentOperation.cs

```diff
@@ -126,30 +126,36 @@ public async Task<int> RunAsync()
         }
     }
 
+    private async Task RemoveRevisionLabel(string revisionName, string label)
+    {
+        var result = await InvokeAzCLI([
+                "containerapp", "revision", "label", "remove",
+                ],
+            [
+                "--label", label
+            ]);
+        result.ThrowIfFailed($"Failed to remove label {label} from revision {revisionName}. Stderr: {result.StandardError}");
```

Comment:
Isn't the stderr printed in `ThrowIfFailed`?

Reply:
Ah you're right, it prints it out in the `ToString` override